### PR TITLE
Fix repository field generation on Windows

### PR DESCRIPTION
**Pull request by joshwooding**

### CODE REVIEW

File: packages/modular-scripts/src/build/buildPackage/getRepositoryField.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ async function getRelativePathInRepo(packagePath: string) {
     ['rev-parse', '--show-toplevel'],
     { stdout: 'pipe' },
   );
-  return path.relative(gitRepoPath, packagePath);
+  return path.relative(gitRepoPath, packagePath).replace(/\\/g, '/');
```
Comment (cristiano-belloni) [inline packages/modular-scripts/src/build/buildPackage/getRepositoryField.ts:14]: Can you please use `path.win32.sep` and `path.posix.sep`?

**Review by cristiano-belloni — CHANGES_REQUESTED** [review]
It seems node 14 is failing with `path.relative(...).replaceAll is not a function`.
This is because we transpile, but not polyfill.

File: packages/modular-scripts/src/build/buildPackage/getRepositoryField.ts [hunk 2]
```diff
@@ -11,7 +11,10 @@ async function getRelativePathInRepo(packagePath: string) {
     ['rev-parse', '--show-toplevel'],
     { stdout: 'pipe' },
   );
-  return path.relative(gitRepoPath, packagePath);
+  return path
+    .relative(gitRepoPath, packagePath)
+    .split(path.win32.sep)
```
Comment (joshwooding) [inline packages/modular-scripts/src/build/buildPackage/getRepositoryField.ts:16]: This is the same as the utils in esbuild-scripts

**Comment by joshwooding** [comment]
Any update on this?

**Review by cristiano-belloni — APPROVED** [review]

**Pull request merged**